specs/player_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'simplecov'
+SimpleCov.start
+
+require 'minitest/autorun'
+require 'minitest/reporters'
+# require 'minitest/skip_dsl'
+require 'minitest/pride'
+require_relative '../lib/player'
+
+describe "Player Class" do
+  describe "tests tehut wrote" do
+    describe "Test that Player Class exists" do
+      it "exists" do
+        # skip
+        tofu = Scrabble::Player.new("tofu")
+        tofu.class.must_be_kind_of Class
+      end
+    end
+
+    describe "Player class has a name" do
+      it "Will return a string when .name is called" do
+        # skip
+        tofu = Scrabble::Player.new("tofu")
+        tofu.name.must_be_kind_of String
+      end
+    end
+
+      describe ".plays initializes as an array" do
+        it "is an array upon instantiation" do
+          # skip
+          tofu = Scrabble::Player.new("tofu")
+          tofu.plays.must_be_kind_of Array
+        end
+      end
+
+
+    describe "Test the .play Method" do
+      describe "Test play method returns an integer" do
```

Comment:
In general, you don't need to prefix your test names with "Test" - we already know that you're testing something. Try and make it short and sweet - what are you `describe`-ing?